### PR TITLE
[HVR] Do not close the OpenXR session when exiting the security zone

### PR DESCRIPTION
**Pull request by svillar**

### CODE REVIEW

**Comment by svillar** [comment]
This will actually fix #49

**Comment by svillar** [comment]
Also #50. That is not a Wolvic issue but a problem in the SDK

File: app/src/hvr/cpp/native-lib.cpp [hunk 1]
```diff
@@ -116,12 +123,15 @@ extern "C" {
 
 JNI_METHOD(void, nativeOnCreate)
 (JNIEnv *aEnv, jobject activity) {
+  LOGI("oncreate");
```
Comment (elima) [inline app/src/hvr/cpp/native-lib.cpp:126]: Perhaps use `onCreate` (in camelCase) for consistency with the other similar logs.

File: app/src/hvr/cpp/native-lib.cpp [hunk 2]
```diff
@@ -25,6 +26,8 @@
   JNIEXPORT return_type JNICALL              \
     Java_com_igalia_wolvic_PlatformActivity_##method_name
 
+#define LOGI(...) __android_log_print(ANDROID_LOG_INFO, "nativeLib", __VA_ARGS__)
```
Comment (elima) [inline app/src/hvr/cpp/native-lib.cpp:29]: Shouldn't we use `LOGD` and correspondingly `ANDROID_LOG_DEBUG`?
The logs being added seem to be for debugging purposes, not very useful as general app info.

**Review by elima — APPROVED** [review]
Apart from the two nitpicks I left, looks good. Feel free to address it (or not) before merging.

File: app/src/hvr/cpp/native-lib.cpp (hunk 2 above)
Comment (svillar) [reply app/src/hvr/cpp/native-lib.cpp:29]: Makes sense

File: app/src/hvr/cpp/native-lib.cpp (hunk 1 above)
Comment (svillar) [reply app/src/hvr/cpp/native-lib.cpp:126]: Sure

**Pull request merged**